src/Ui/Action/Admin/ResendInvoiceAction.php

```diff
@@ -29,9 +29,10 @@ final class ResendInvoiceAction
     /** @var UrlGeneratorInterface */
     private $urlGenerator;
 
-    /** @var Session */
+    /** @var Session|Session[] */
```

Comment:
I'm not conviced to that, as the `$session` argument has `Session` type. Maybe we should exclude that, wdyt @pamil @Zales0123 @lchrusciel

Reply:
It's quite strange such a change is required... You're sure it's not working without it 😄 if no, I would vote to exclude this check and open an issue to investigate it properly

Reply:
I excluded it: https://github.com/Sylius/InvoicingPlugin/pull/169/files#diff-e4ce1df0d2ca17a9d91544a50ace127aR13-R14